Docs/source/usage/parameters.rst

```diff
@@ -1395,6 +1401,53 @@ External fields
       and :math:`E_z = 0`, and
       :math:`B_x = \mathrm{strength} \cdot y`, :math:`B_y = -\mathrm{strength} \cdot x`, and :math:`B_z = 0`.
 
+Accelerator Lattice
+^^^^^^^^^^^^^^^^^^^
+
+Several accelerator lattice elements can be defined as described below.
+The elements are defined relative to the `z` axis and in the lab frame. They are described using a simplified MAD like syntax.
```

Comment:
maybe it could help to clarify:  "... defined relative to the `z` axis starting at `z=0`. ..."

Reply:
Ok, I'll add this